system_tests/Dockerfile

```diff
@@ -16,6 +16,10 @@ COPY scripts/ci/* ./ci/
 COPY datastore datastore
 COPY tests tests
 
+RUN mkdir -p /run/secrets
+RUN echo -n openslides >/run/secrets/datastore_postgres_user
+RUN echo -n openslides >/run/secrets/datastore_postgres_password
+
```

Comment:
The first tests are failing, because you only added these lines to `/system_tests/Dockerfile`. However, the prod setup tries to read the credentials from `/Dockerfile`. Please let us talk about this.